README.md

```diff
@@ -124,6 +184,9 @@ For more information on the Zendesk Apps Tools please see the [documentation](ht
 ## External Dependencies
 External dependencies are defined in the  [`webpack.config.js`](https://github.com/zendesk/app_scaffold/blob/master/webpack.config.js) as `externalAssets`. This ensures these dependencies are included on your app's `index.html` as well as in the test suite.
 
+## Owners
+The project is maintained by the Quokka / Vegemite team, ping us with [@zendesk/vegemite on GitHub](https://github.com/orgs/zendesk/teams/vegemite).
```

Comment:
I think this is a no-go for open source repos, since these handles don't work for most people?